x-pack/plugins/security_solution/public/common/components/data_table/index.tsx

```diff
@@ -74,11 +77,18 @@ export interface DataTableProps {
   unitCountText: string;
   pagination: EuiDataGridPaginationProps;
   totalItems: number;
+  rowHeightsOptions?: EuiDataGridRowHeightsOptions;
+  isEventRenderedView?: boolean;
 }
 
 const ES_LIMIT_COUNT = 9999;
 
-const gridStyle: EuiDataGridStyle = { border: 'none', fontSize: 's', header: 'underline' };
+const gridStyle = (isEventRenderedView: boolean | undefined = false): EuiDataGridStyle => ({
+  border: 'none',
+  fontSize: 's',
+  header: 'underline',
+  stripes: isEventRenderedView === true,
```

Comment:
Not sure about the benefits to have the `stripes` for 'eventRenderedView' only at least from the design prospective. Is it because the blurs transition between the wide summary description for each row?

Reply:
Maybe? @XavierM or @andrew-goldstein may be able to answer